code/components/citizen-server-impl/src/state/ServerGameState.cpp

```diff
@@ -6756,6 +6758,14 @@ std::function<bool()> fx::ServerGameState::GetGameEventHandler(const fx::ClientS
 		return GetRequestControlEventHandler(client, std::move(buffer));
 	}
 
+	if(eventType == NETWORK_PLAY_SOUND_EVENT)
+	{
+		return []()
+		{
+			return g_networkedSoundsEnabledVar->GetValue();
```

Comment:
Should this not be a static bool (see other convars here for reference) instead of using `GetValue()`